tests/test_numpy.py

```diff
@@ -1,17 +1,19 @@
 from tensorboardX import x2num
 import numpy as np
+
+
 def test_scalar():
-    res = x2num.makenp(1.1)
+    res = x2num.make_np(1.1)
```

Comment:
should be another PR.
Lets separate into:
- histogram fix
- funcation name change
- visdom
- caffe2
- fix test